content/building-blocks/prepare-your-data-for-analysis/data-preparation/large-datasets-R.md

```diff
@@ -25,19 +29,19 @@ In addition, switching from the `read.csv()` function to `fread()` can greatly i
 library(data.table)
 
 # import data with data.table package
-df <- fread(<YOUR_DATASET.csv>)
+df <- fread("YOUR_DATASET.csv")
 
 # only import the first couple of rows for exploratory analysis 
-df <- fread(<YOUR_DATASET.csv>, nrows=500)
+df <- fread("YOUR_DATASET.csv", nrows=500)
 
 # only import the data you actually use 
-df <- fread(<YOUR_DATASET.csv>, select=c(1, 2, 5))  # column indices
-df <- fread(<YOUR_DATASET.csv>, select=c("date", "country", "revenue"))  # column names
+df <- fread("YOUR_DATASET.csv", select=c(1, 2, 5))  # column indices
+df <- fread("YOUR_DATASET.csv", select=c("date", "country", "revenue"))  # column names
 
 # print object size in bytes (for a quick comparison)
 object.size(df)
 
 # store the derivative file for future use
-fwrite(df, <YOUR_CLEANED_DATSET.csv>)
+fwrite(df, "YOUR_CLEANED_DATSET.csv")
```

Comment:
type: DATASET*

Reply:
Ooops, Thanks for pointing that out Roshini! Typo corrected! ⭐